graph/adjacency/impl/ThingAdjacencyImpl.java

```diff
@@ -129,47 +135,55 @@ public ThingEdge edge(Encoding.Edge.Thing encoding, ThingVertex adjacent) {
         return edge;
     }
 
-    private ThingEdgeImpl put(Encoding.Edge.Thing encoding, ThingEdgeImpl edge, IID[] infixes, boolean isModified, boolean recurse) {
+    private ThingEdgeImpl put(Encoding.Edge.Thing encoding, ThingEdgeImpl edge, IID[] infixes, boolean isModified, boolean symmetricPut) {
```

Comment:
renamed "recurse" to "symmetricPut" as "recursive" makes the user think it can go arbitrarily deep!

Reply:
name this to `putReflexive`